lab-examples/clos02/README.md

```diff
@@ -0,0 +1,10 @@
+# SRL Containerlab (clos02)
```

Comment:
Thanks @siva19susi 

can you please give here a small explanation as to what is being configured with those config files

i.e. which protocols between which DC layers, etc
I will copy it then to the examples description so that it will get rendered in the docs site

Reply:
Sure. Will add them through.